experiments/pages/lightningModalLauncher/lightningModalLauncher.xml

```diff
@@ -0,0 +1,202 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true" theme="Lightning">
+	<models>
+		<model id="Contact" limit="1" query="true" createrowifnonefound="false" datasource="salesforce" sobject="Contact">
+			<fields>
+				<field id="Id"/>
+				<field id="Name"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="FlowParams" limit="20" query="true" datasource="Ui-Only" createrowifnonefound="true">
+			<fields>
+				<field id="name" displaytype="TEXT" length="255"/>
+				<field id="type" displaytype="TEXT" length="255"/>
+				<field id="value" displaytype="TEXT" length="255"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="Flow" limit="20" query="true" datasource="salesforce" sobject="FlowInterview">
+			<fields>
+				<field id="Name"/>
+			</fields>
+			<conditions>
+				<condition type="fieldvalue" value="" enclosevalueinquotes="true" field="Name" operator="=" clientorserver="server" state="filterableon" inactive="false" name="Name"/>
+			</conditions>
+			<actions/>
+		</model>
+	</models>
+	<components>
+		<skuid__buttonSet uniqueid="sk-1c1D-18257" position="center" model="Contact">
+			<groups>
+				<skuid__buttonGroup displayType="detached">
+					<buttons>
+						<skuid__button label="Launch Lightning Modal" uniqueId="civilianTrainingButton">
+							<actions>
+								<action type="emptyModelData">
+									<models>
+										<model>FlowParams</model>
+									</models>
+								</action>
+								<action type="createRow" model="FlowParams" appendorprepend="prepend" defaultmodefornewitems="edit" affectedrows="context" enabled="false">
+									<defaults>
+										<default valuesource="fieldvalue" field="name" enclosevalueinquotes="true" value="cid"/>
+										<default valuesource="fieldvalue" field="type" enclosevalueinquotes="true" value="String"/>
+										<default valuesource="modelmerge" field="value" enclosevalueinquotes="true" sourcemodel="Contact" sourcefield="Id"/>
+									</defaults>
+								</action>
+								<action type="action-sequence" action-sequence-id="29f52636-5af5-49b1-8550-54170e17e336">
+									<inputs>
+										<input name="model" value="FlowParams"/>
+										<input name="flowName" value="HelloWorldFlow"/>
+									</inputs>
+								</action>
+							</actions>
+							<tooltip position="top"/>
+							<renderConditions logictype="or"/>
+							<enableConditions logictype="and"/>
+							<styleVariantConditions/>
+						</skuid__button>
+					</buttons>
+				</skuid__buttonGroup>
+			</groups>
+			<styles>
+				<spacing bottom="2"/>
+			</styles>
+			<renderConditions logictype="and"/>
+			<styleVariantConditions/>
+		</skuid__buttonSet>
+	</components>
+	<resources>
+		<labels/>
+		<javascript>
+			<jsitem location="inline" name="EventHotFix" cachelocation="false" url="">((skuid) =&gt; {
```

Comment:
I believe this was fixed in `Boston (13.0.12)` and later versions (CORE-3239), and this is no longer needed as a result, so I would remove it and note that this example requires you to be on at least `Boston (13.0.12)`.